### PR TITLE
Shadowling balance #2

### PR DESCRIPTION
**Pull request by ghost**
:cl: Thefastfoodguy
balance: Slings take less damage from red light
balance: Slings take slightly less damage from green light
tweak: You no longer need the special dethralling surgery to dethrall people, normal organ manipulation is fine
fix: Thrall darksight now works, after almost half the year!
fix: Ascendant Shadowlings Lightning Storm now actually works
add: Ascendant Shadowlings now have the Gore ability, which allows them to instantly dash into and gore an enemy.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
WOOO FIXED THRALL DARKSIGHT ONCE AND FOR ALL

**Comment by Blute1** [comment]
he's back to normal yay

**Review by GuyonBroadway — DISMISSED** [review]
Code looks ok

**Comment by ghost** [comment]
@HippieStation/head Please +1 this

@HippieStation/maintainers please approve/review this

File: hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm [hunk 1]
```diff
@@ -36,8 +36,8 @@
 
 
 /obj/effect/proc_holder/spell/targeted/sling/InterceptClickOn(mob/living/caller, params, atom/t)
-	if(!ishuman(t))
-		to_chat(caller, "<span class='warning'>You may only use this ability on humans!</span>")
+	if(!isliving(t))
+		to_chat(caller, "<span class='warning'>You may only use this ability on living things!</span>")
```
Comment (ghost) [inline hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm:40]: Now you can gib monkeys as an Ascendant! Hooray!

**Comment by dreamcrab** [comment]
slings kinda just get fucked up without jaunt
were they deemed too powerful without it?

**Comment by ghost** [comment]
@dreamcrab What jaunt? Jaunt got replaced by the passive shadowwalk ability that allows them to move really fast through darkness and phase through walls (if it's dark on the opposite side), with the downside of being slow in light

**Comment by ghost** [comment]
cool ok +1

**Comment by ghost** [comment]
but yeah this makes thrall darksight FINALLY WORK

**Comment by dreamcrab** [comment]
oh okay ive only seen shadowlings once since they got changed and they all died really fast

**Comment by ghost** [comment]
Tbh if you don't know the mechanics of shadowlings, you WILL die really fast. You know what... ima just make it so you need to have played for 2 weeks to be a sling

**Comment by ghost** [comment]
@Chnkr is the shitty commit history due to me, or a general git repo fuckup? I tried to pull from master, but it said I was up to date

**Comment by dreamcrab** [comment]
dont make a whitelist or i wont approve it

**Comment by ghost** [comment]
+1 is retracted if there is whitelist

**Comment by ghost** [comment]
Your PR needs to be updated to reflect the current state of the master branch, please update your PR and either (re)open it, or ask a maintainer to (re)open it.

**Comment by ghost** [comment]
@HippieStation/maintainers Please reopen this, i'm trying to rebase right now (git doesn't let me see if it actually worked if this pr is closed)

**Comment by ghost** [comment]
@HippieStation/head There is no whitelist, i'm just upping the time to unlock shadowling to 2 weeks, like headrevs, cult, nukeops, etc

**Comment by ghost** [comment]
we're not whitelisting antag roles

**Comment by ghost** [comment]
https://i.imgur.com/WCYS5ZT.png

I can't reopen this.

**Comment by ghost** [comment]
ohk

**Pull request closed without merging**